content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -27,7 +27,7 @@ answer the following questions:
 
 {{% capture body %}}
 
-[Kube-apiserver][kube-apiserver] performs auditing. Each request on each stage
+[Kube-apiserver](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/) performs auditing. Each request on each stage
```

Comment:
this change is incorrect. The link address is found in line 363. The original syntax is correct.